cdc/log.cc

```diff
@@ -993,24 +996,36 @@ cdc::cdc_service::impl::augment_mutation_call(lowres_clock::time_point timeout,
                 // Note: further improvement here would be to coalesce the pre-image selects into one
                 // iff a batch contains several modifications to the same table. Otoh, batch is rare(?)
                 // so this is premature.
+                tracing::trace(tr_state, "CDC: Selecting preimage, {}", m.decorated_key());
```

Comment:
s/Selecting preimage, {}/Selecting preimage for {}/

There are similar messages below. It is confusing to see the key after the comma.

Reply:
Done, for other similar messages as well